### DISCUSSION TITLE
multi tracks on map

### DISCUSSION THREAD

**Discussion by dizmoduck** [OpenTracksApp/OpenTracks#1504] [Ideas]
it would be nice if you could choose several tracks on the same map? 

I tested 5-10 defeated tracking app after I had use GeoTracker for meny years and this one I found to be the best

**Comment by pstorch** [comment]
Select multiple tracks in OpenTracks and show them on the Map of OSMDashboard is already possible.

**Reply by dizmoduck** [reply]
Ups i blame that alzheimer lite.  I thought I had tried try that

thank
